src/storage/exec/ScanNode.h

```diff
@@ -262,9 +274,16 @@ class ScanEdgePropNode : public QueryNode<Cursor> {
     return nebula::cpp2::ErrorCode::SUCCEEDED;
   }
 
-  void collectOneRow(bool isIntId, std::size_t vIdLen) {
+  nebula::cpp2::ErrorCode collectOneRow(bool isIntId, std::size_t vIdLen) {
     List row;
     nebula::cpp2::ErrorCode ret = nebula::cpp2::ErrorCode::SUCCEEDED;
+    // Usually there is only one edge node, when all of the egdeNodes are invalid (e.g. ttl
+    // expired), just skip the row. If we don't skip it, there will be a whole line of empty value.
```

Comment:
In fact, we should return one only if vertex key exists.

Reply:
Where did you handle this in scan before? BTW, the vertex key will be removed later

Reply:
When introducing this, there is no vertex key.

Reply:
It's a bug delayed https://github.com/vesoft-inc/nebula/issues/3688